### PR TITLE
new package: stuntman

### LINKED ISSUE #5191: Package Request: stuntman-client
Package: stuntman-client (1.2.7-1.1)
Test client for STUN (Session Traversal Utilities for NAT)

http://manpages.ubuntu.com/manpages/cosmic/man1/stunclient.1.html

### PR DESCRIPTION
**Pull request by xtkoba**
Closes #5191.

### CODE REVIEW

File: packages/stuntman/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="T="
+
+termux_step_configure() {
+	:
+}
+
+termux_step_make_install() {
+	install -Dm700 -t $TERMUX_PREFIX/bin "$TERMUX_PKG_BUILDDIR/stunclient"
+	install -Dm700 -t $TERMUX_PREFIX/sbin "$TERMUX_PKG_BUILDDIR/stunserver"
```
Comment (Grimler91) [inline packages/stuntman/build.sh:19]: Please install to /bin

File: packages/stuntman/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+TERMUX_PKG_SRCURL=http://www.stunprotocol.org/stunserver-${TERMUX_PKG_VERSION}.tgz
+TERMUX_PKG_SHA256=4479e1ae070651dfc4836a998267c7ac2fba4f011abcfdca3b8ccd7736d4fd26
+TERMUX_PKG_DEPENDS="libc++, openssl"
+TERMUX_PKG_BUILD_DEPENDS="boost"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="T="
+
+termux_step_configure() {
+	:
+}
```
Comment (Grimler91) [inline packages/stuntman/build.sh:15]: Should be no need to override termux_step_configure

File: packages/stuntman/build.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+TERMUX_PKG_DESCRIPTION="An open source STUN server"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.2.16
+TERMUX_PKG_SRCURL=http://www.stunprotocol.org/stunserver-${TERMUX_PKG_VERSION}.tgz
+TERMUX_PKG_SHA256=4479e1ae070651dfc4836a998267c7ac2fba4f011abcfdca3b8ccd7736d4fd26
+TERMUX_PKG_DEPENDS="libc++, openssl"
+TERMUX_PKG_BUILD_DEPENDS="boost"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="T="
```
Comment (Grimler91) [inline packages/stuntman/build.sh:11]: Is T= necessary? Seem to work fine without it when I test build

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/stuntman/build.sh (hunk 1 above)
Comment (xtkoba) [reply packages/stuntman/build.sh]: Fixed.

File: packages/stuntman/build.sh (hunk 2 above)
Comment (xtkoba) [reply packages/stuntman/build.sh]: Fixed.

File: packages/stuntman/build.sh (hunk 3 above)
Comment (xtkoba) [reply packages/stuntman/build.sh]: May be needed if environment variable `T` is defined. A comment is added. Thanks!

**Pull request merged**